src/modules/shared.mjs

```diff
@@ -105,92 +76,227 @@ export function addFeedbackLink() {
   document.body.appendChild(aside);
 }
 
-export function updateLogo() {
-  const logo = document.getElementsByTagName("img")[0];
-  logo.src = "https://beta.nj.gov/files/dol_logo.png";
-  logo.style.removeProperty("width");
-  logo.style.removeProperty("height");
-  logo.style.marginLeft = isDesktop() ? "107px" : "18px";
-  logo.alt = "Official logo for the New Jersey Department of Labor";
-  logo.parentElement.parentElement.removeAttribute("height");
+// Used for tagged template strings
+export function html(strings, ...values) {
+  return String.raw({ raw: strings }, ...values);
 }
 
-// Used for tagged template strings
-const html = (strings, ...values) => String.raw({ raw: strings }, ...values);
+export function js(strings, ...values) {
+  return String.raw({ raw: strings }, ...values);
+}
+
+export function css(strings, ...values) {
+  return String.raw({ raw: strings }, ...values);
+}
 
 export const HEADER_HTML = html`
   <header
-    style="padding:12px 18px;background-color:black;color:white;font-size:13px;line-height:150%;margin-bottom:52px"
+    style="padding: 10px 20px; background-color: black; color: white; font-size: 13px; line-height: 150%"
```

Comment:
Dust: I kind of want consistency with hex codes for colors, but also it really is that not consequential haha

Reply:
I'm with you, I'll change these out, it feels more explicit